# message containing darpa's reply after artifact submission
string unique_id
float32 submission_time_raw
string artifact_type
float32 x
float32 y
float32 z
string report_status
uint8 score_change
string http_response
string http_reason

 